views/layout.erb

```diff
@@ -10,82 +10,75 @@
   <meta name="description" content="Custom BLAST server provided by SequenceServer (https://sequenceserver.com)"/>
 
   <%= Rack::Csrf.metatag(env) %>
-  <!-- CSS -->
-  <% if settings.production? || settings.test? %>
-    <link rel="stylesheet" media="screen,print" type="text/css" href="css/sequenceserver.min.css?ver=<%= SequenceServer::VERSION %>"/>
-  <% else %>
-    <link rel="stylesheet" media="screen,print" type="text/css" href="css/sequenceserver.css"/>
-  <% end %>
+  <link rel="stylesheet" media="screen,print" type="text/css" href="css/app.min.css"/>
```

Comment:
I'd say keep the `?ver=<%= SequenceServer::VERSION %>"` to bust browser cache between gem releases

Reply:
updated